docker-compose.yml

```diff
@@ -156,6 +156,100 @@ services:
       cp /root/.lnd/macaroons.db /root/.lnd/data/chain/bitcoin/regtest/macaroons.db
       cp /root/.lnd/admin.macaroon /root/.lnd/data/chain/bitcoin/regtest/admin.macaroon
       /bin/lnd
+  loopserver:
+    image: lightninglabs/loopserver:v0.9.52-beta
+    restart: unless-stopped
+    volumes:
+      - ${PWD}/dev/lnd/lnd.conf:/root/.lnd/lnd.conf
+      - ${PWD}/dev/lnd/tls.key:/root/.lnd/tls.key
+      - ${PWD}/dev/lnd/tls.cert:/root/.lnd/tls.cert
+      - ${PWD}/dev/lnd/regtest/lnd-outside-1.admin.macaroon:/root/.lnd/data/chain/bitcoin/regtest/admin.macaroon
+      - ${PWD}/dev/lnd/regtest/lnd-outside-1.admin.macaroon:/root/.lnd/data/chain/bitcoin/regtest/readonly.macaroon
+      - ${PWD}/dev/lnd/regtest/lnd-outside-1.admin.macaroon:/root/.lnd/data/chain/bitcoin/regtest/invoices.macaroon
+      - ${PWD}/dev/lnd/regtest/lnd-outside-1.admin.macaroon:/root/.lnd/data/chain/bitcoin/regtest/signer.macaroon
+      - ${PWD}/dev/lnd/regtest/lnd-outside-1.admin.macaroon:/root/.lnd/data/chain/bitcoin/regtest/chainnotifier.macaroon
+      - ${PWD}/dev/lnd/regtest/lnd-outside-1.admin.macaroon:/root/.lnd/data/chain/bitcoin/regtest/router.macaroon
+      - ${PWD}/dev/lnd/regtest/lnd-outside-1.admin.macaroon:/root/.lnd/data/chain/bitcoin/regtest/walletkit.macaroon
+    depends_on:
+      - lnd-outside-1
+    ports:
+      - "11009:11009"
```

Comment:
for some reason ports are included in the docker-compose override files (not sure why that is, @bodymindarts ?)

Reply:
yes, i will move them to override. maybe to make it easier to override if we get a port conflict